config/environments.rb

```diff
@@ -0,0 +1,14 @@
+# The environment variable DATABASE_URL should be in the following format:
+# => postgres://{user}:{password}@{host}:{port}/path
+configure :production, :development do
```

Comment:
Why do you need this file at all? I'm sure that there are much easier ways to connect to DB